custom-nginx-reverse-proxy.conf

```diff
@@ -83,13 +101,17 @@ http {
             add_header X-debug "/api/v1/streams";
             proxy_pass http://brokers_http;
             proxy_read_timeout 240s;
+            proxy_http_version 1.1;
+            proxy_set_header Connection "";
         }
 
         # Data Volume endpoint
         location /api/v1/volume {
             add_header X-debug "/api/v1/volume";
             proxy_pass http://brokers_http;
             proxy_read_timeout 240s;
+            proxy_http_version 1.1;
+            proxy_set_header Connection "";
```

Comment:
https://www.nginx.com/blog/tuning-nginx/ Search for "Keepalive Connections". Nginx default HTTP protocol is 1.0. HTTP/1.1 has keepalive (HTTP not TCP) connections. Nginx can cache connections for better performance. I intentionally used a low value for keepalive connections (it can be increased if needed):
```
     upstream eae {
          server 10.200.10.1:8081;
+         keepalive 20;
+         keepalive_requests 50;
+         keepalive_timeout 60s;
      }
```